bokehjs/src/coffee/main.coffee

```diff
@@ -1,15 +1,10 @@
-Bokeh = {}
-Bokeh.require = require
-Bokeh.version = '0.9.2'
```

Comment:
I should have added a banner here, saying "don't touch unless you increment version number".